components/table/test/table.vdiff.js

```diff
@@ -0,0 +1,1229 @@
+												<th style="white-space: nowrap">Item 5</th>
+											</tr>
+										</thead>
+										<tbody>
+											<tr>
+												<th sticky style="white-space: nowrap; z-index: 2">
+													Albert, Eddie
+													<span style="position: relative;">
+														&nbsp;&#8964;
+														<div style="background-color: blue; color: white; position: absolute; top: 20px; left: 0px; padding: 10px; width: 145px; height: 175px;">Dropdown simulator</div>
+													</span>
+												</th>
+												<td>5/10</td>
+												<td>9/10</td>
+												<td>3/5</td>
+												<td>9/10</td>
+												<td class="over">9/10</td>
+											</tr>
+											<tr>
+												<th sticky style="white-space: nowrap">Bedelia, Bonnie</th>
+												<td>3/10</td>
+												<td>7/10</td>
+												<td>5/5</td>
+												<td>7/10</td>
+												<td>7/10</td>
+											</tr>
+											<tr>
+												<th sticky style="white-space: nowrap">Benson, Robbie</th>
+												<td>8/10</td>
+												<td>6/10</td>
+												<td>1/5</td>
+												<td>6/10</td>
+												<td>6/10</td>
+											</tr>
+										</tbody>
+									</table>
+								`
+							}
+						].forEach(({ name, template }) => {
+							['top', 'over'].forEach(position => {
+								it(`${name}-${position}`, async() => {
+									const elem = await createFixture(template, { bottomMargin: true, stickyHeaders: true });
+									elem.shadowRoot.querySelector(`.${position}`).scrollIntoView(true);
+									await expect(elem).to.be.golden({ margin: 0 });
+								});
+							});
+						});
+
+					});
+
+					describe('table-controls', () => {
+
+						[
+							{ name: 'no-sticky', stickyControls: false, stickyHeaders: false, visibleBackground: false },
+							{ name: 'sticky-controls', stickyControls: true, stickyHeaders: false, visibleBackground: false },
+							{ name: 'all-sticky', stickyControls: true, stickyHeaders: true, visibleBackground: false },
+							{ name: 'visible-background', stickyControls: true, stickyHeaders: true, visibleBackground: true },
+						].forEach(condition1 => {
+							describe(condition1.name, () => {
+
+								[
+									{ name: '1-top', scroll: 0 },
+									{ name: '2-scrolled', scroll: 1000 },
+								].forEach(condition2 => {
+									it(condition2.name, async() => {
+										const elem = await fixture(
+											html`
+												<div style="height: 300px;overflow-y: scroll;">
+													<d2l-test-table
+														condensed
+														?sticky-controls="${condition1.stickyControls}"
+														?sticky-headers="${condition1.stickyHeaders}"
+														type="${type}"
+														?visible-background="${condition1.visibleBackground}">
+													</d2l-test-table>
+												</div>
+											`,
+											{ rtl: dir === 'rtl', viewport: { width: 500 } }
+										);
+										elem.scrollTo(0, condition2.scroll);
+										await nextFrame();
```

Comment:
These are the ones where I'm seeing flake with the scroll wrapper dashed borders. The `nextFrame` was a failed attempt to fix them.